_data/investing.yml

```diff
@@ -29,6 +29,7 @@ websites:
     url: https://www.schwab.com/
     img: charlesschwab.png
     tfa:
+      - sms
```

Comment:
I don't see any mentions of SMS in the note?

Reply:
Thanks @Carlgo11. Agreed...I added the SMS flag at the last moment after noticing that it wasn't there (yet having used it personally for several years)...thus, didn't think about adding it to the note. I have done so now _and_ explicitly confirmed Symantec support by Fidelity.